.eslintrc

```diff
@@ -82,6 +82,22 @@
         "no-useless-constructor": "off",
         "import-name": [true, { "react": "React" }]
       }
+    },
+    {
+      "plugins": ["es"],
+      "files": [
+        "**/packages/reakit/**/*",
+        "**/packages/reakit-system-bootstrap/**/*",
+        "**/packages/reakit-system-palette/**/*"
+      ],
+      "parserOptions": {
+        "ecmaVersion": 2018
+      },
+      "excludedFiles": ["**/__tests__/**/*"],
+      "rules": {
+        "es/no-async-functions": "error",
+        "es/no-async-iteration": "error"
+      }
```

Comment:
I think the added complexity doesn't worth the minimal gain it provides. We can spot the rare cases when async/await is used on code review.

Reply:
Okay, I'll revert that

Reply:
DOne